### ISSUE TITLE
BUG: pd.read_clipboard() not working in WSL2

### ISSUE THREAD

**Issue by pandichef** [pandas-dev/pandas#39379]
- [x] I have checked that this issue has not already been reported.

- [x] I have confirmed this bug exists on the latest version of pandas.

- [ ] (optional) I have confirmed this bug exists on the master branch of pandas.

---
#### Code Sample, a copy-pastable example
```python
pd.read_clipboard()
```

#### Problem description
I recently started using WSL2 on Windows 10 Pro / Ubuntu 20.04 / Python 3.8.5.  In general, It works incredibly well today (unlike in 2018).  Unfortunately, pd.read_clipboard() and df.to_clipboard() don't work.  I believe the problem is pandas because I got the desired functionality to work using pyperclip directly.

#### Error Message
>>> pd.read_clipboard()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mickeymouse/.local/lib/python3.8/site-packages/pandas/io/clipboards.py", line 38, in read_clipboard
    text = clipboard_get()
  File "/home/mickeymouse/.local/lib/python3.8/site-packages/pandas/io/clipboard/__init__.py", line 649, in lazy_load_stub_paste
    return paste()
  File "/home/mickeymouse/.local/lib/python3.8/site-packages/pandas/io/clipboard/__init__.py", line 287, in __call__
    raise PyperclipException(EXCEPT_MSG)
pandas.io.clipboard.PyperclipException:
    Pyperclip could not find a copy/paste mechanism for your system.
    For more information, please visit
    https://pyperclip.readthedocs.io/en/latest/introduction.html#not-implemented-error

**Comment by MarcoGorelli** [comment]
Hi @pandichef 

Can you show the output of `pandas.show_versions()` (as per the issue template)? I thought this would've been fixed in #38546

**Comment by pandichef** [comment]
Sorry about that.
```
INSTALLED VERSIONS
------------------
commit           : 3e89b4c4b1580aa890023fc550774e63d499da25
python           : 3.8.5.final.0
python-bits      : 64
OS               : Linux
OS-release       : 4.19.128-microsoft-standard
Version          : #1 SMP Tue Jun 23 12:58:10 UTC 2020
machine          : x86_64
processor        : x86_64
byteorder        : little
LC_ALL           : None
LANG             : C.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.2.0
numpy            : 1.19.5
pytz             : 2020.5
dateutil         : 2.8.1
pip              : 20.0.2
setuptools       : 45.2.0
Cython           : None
pytest           : None
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : None
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : 2.10.1
IPython          : 7.19.0
pandas_datareader: None
bs4              : None
bottleneck       : None
fsspec           : None
fastparquet      : None
gcsfs            : None
matplotlib       : 3.3.3
numexpr          : None
odfpy            : None
openpyxl         : None
pandas_gbq       : None
pyarrow          : None
pyxlsb           : None
s3fs             : None
scipy            : 1.6.0
sqlalchemy       : None
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
xlwt             : None
numba            : None
```

**Comment by MarcoGorelli** [comment]
OK, thanks - looks like the fix will be in 1.3.0, so closing this as a duplicate of #38527
